src/main/java/gov/acwi/wqp/etl/DbConfig.java

```diff
@@ -14,41 +14,39 @@
 public class DbConfig {
 
 	@Bean
-	@ConfigurationProperties("spring.datasource-wqp")
+	@ConfigurationProperties(prefix="spring.datasource-wqp")
 	@Primary
 	@Profile("default")
-	public DataSourceProperties wqpDataSourceProperties() {
+	public DataSourceProperties dataSourceWqpProperties() {
 		return new DataSourceProperties();
 	}
 	
 	@Bean
-	@ConfigurationProperties("spring.datasource-wqp")
 	@Primary
 	@Profile("default")
-	public DataSource wqpDataSource() {
-		return wqpDataSourceProperties().initializeDataSourceBuilder().build();
+	public DataSource dataSourceWqp() {
+		return dataSourceWqpProperties().initializeDataSourceBuilder().build();
```

Comment:
This might need to be "dataSourcePropertiesWqp"

Reply:
Updated